templates/Notifications/NotificationTypes/achievement_notification.html.twig

```diff
@@ -1,33 +1,16 @@
-<div class="card mb-3 notification-container" id="catro-notification-{{ notification.id }}">
+{% extends 'Notifications/NotificationTypes/notification_items.html.twig' %}
 
-  <h5 class="card-header notification-header">
-    {{ notification.title }}
-  </h5>
-  <div class="card-body">
-
-    <div class="row">
-
-      <div class="notification-avatar">
-        <img class="notification-round"
-             src="{{ notification.getImagePath() }}"
-             alt="Card image">
-      </div>
-
-      <div class="notification-body">
-
-        <div class="row mb-4">
-          <p class="card-text">
-            {{ notification.message }}
-        </div>
+{% block notification_leftAvatar %}
+  <div class="col-2 col-sm-1 my-auto">
+    <img class="img-fluid round"
+         src="{{ notification.getImagePath() }}"
```

Comment:
Something to consider for all notifications. 
It would be great if there would be a dummy image. Could even be just a css circle in the primary color as backup behind the image. In case the image is missing

Right now there is just a hole

![image](https://user-images.githubusercontent.com/40868718/89382271-df047200-d6fa-11ea-98d3-748635716b68.png)